iso_week_test.go

```diff
@@ -4,11 +4,17 @@ import (
 	"testing"
 	"time"
 
-	"bitbucket.org/splice/api/models"
+	"bitbucket.org/splice/api/splice/logger"
```

Comment:
Sweet! CI is a good idea. I think I missed the models dependency the first time around.